packages/python/readme_metrics/PayloadBuilder.py

```diff
@@ -214,10 +222,20 @@ def _build_request_payload(self, request) -> dict:
         if "Authorization" in headers:
             headers["Authorization"] = mask(headers["Authorization"])
 
+        if hasattr(request, "environ"):
+            http_version = request.environ["SERVER_PROTOCOL"]
+        elif hasattr(request, "scope"):
+            http_version = f'HTTP/{request.scope["http_version"]}'
+        else:
+            http_version = "HTTP/1.1"  # Assume it is default HTTP version
```

Comment:
Is there a good reason to guess it's HTTP/1.1 if there's no value found? Can we just leave it blank?

Reply:
I think it makes sense. It's better if we leave it blank.